### PR TITLE
Adding metadata and logic to display software licenses

### PR DESCRIPTION
**Pull request by Gwend4l**
As suggested in issue #211, here is an attempt to add new metadata to describe software licenses, and some code to display such information.

Note that this is a first sketch, I am not sure at all that the fields I've added are the right ones, and anything/everything can be reworked. Don't hesitate to reject all this :)

Regarding **metadata**, here are the fields I've added:

- `license`, a string to describe the license (e.g. GPLv3, Commercial)
- `libre`, a boolean to state that the plugin contains libre software
- `proprietary`, a boolean to state that that this is proprietary software
- `patented`, a boolean to state that it contains patented techniques

All the information was mostly taken from Wikipedia, RPMs metadata and Official web pages.

Regarding the **code**, it impacts the number of rows used by a plugin description, hence I've added an early check to know whether we need 2 or 3 rows (`has_license` boolean). Then to display the license, it works the following way:

- If only the `libre` boolean is true, it displays "Libre"
- If only the `proprietary` boolean is true, it displays "Proprietary"
- If only the `patented` boolean is true, it displays "Patented"
- In other cases, it displays "Mixed"

Then the license description is given in between parentheses.

Here is what it currently looks like:
capture d ecran de 2015-06-30 16-13-23

### CODE REVIEW

**Comment by Rohan62442** [comment]
Instead of having multiple booleans, wouldn't it be better to have a single string, say *license*, which will display the text?

**Comment by Gwend4l** [comment]
I started by doing that, but then I realised that there are lots of terms used to talk about FOSS software (e.g. free, open source), and that this could lead to something less homogeneous than controlling what to display in the code (e.g. displaying "libre"). It's also a way to change our mind and to decide that "Proprietary" is not the right word, but "Freeware" is (this is just an example), and to change this without editing any json file.

In other words, it makes sure that everything is formatted in a regular way such as:
[Libre|Proprietary|Mixed] (<description>)

Yet you're probably right; in fact I did begun with a simple string. It might still be the best option to have less lines of code and less metadata fields than with such booleans. If you prefer, I can roll back to this simpler solution.

**Comment by satya164** [comment]
Thank you for your effort. I'll try to visually improve it sometime.
